.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+image: golang:1.11
+
+cache:
+  paths:
+    - /apt-cache
+    - /go/src/github.com
+    - /go/src/golang.org
+    - /go/src/gopkg.in
+    - node_modules
+
+variables:
+  DOCKER_HOST: tcp://docker:2375/
+  DOCKER_DRIVER: overlay2
+
+services:
+  - docker:dind
+  - mongo
+
+stages:
+  - test
+
+before_script:
+  - export DOCKER_REPOSITORY="mendersoftware/workflows"
+  - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
+  - export SERVICE_IMAGE=$DOCKER_REPOSITORY:$DOCKER_TAG
+  - export COMMIT_TAG="$CI_COMMIT_REF_SLUG"_"$CI_COMMIT_SHA"
+
+test:static:
+  stage: test
+  dependencies: []
```

Comment:
Here and below: you don't need an empty dependencies list, better to just omit it.